referenceframe/frame_system_test.go

```diff
@@ -546,3 +546,63 @@ func TestFrameSystemToPCD(t *testing.T) {
 		test.That(t, asBytes, test.ShouldEqual, checkAgainst)
 	})
 }
+
+func TestReplaceFrame(t *testing.T) {
+	fs := NewEmptyFrameSystem("test")
+	pose := spatial.NewZeroPose()
+	dims := r3.Vector{1, 1, 1}
+	box, err := spatial.NewBox(pose, dims, "box")
```

Comment:
There isn't really a reason to have this info in these tests.  they can all be instantiated through `NewZeroStaticFrame`